### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by jarmitage** [shawnlawson/liveware#1]
Interested to try running this but can't figure out how.

I have Lua 5.3.4 installed but I do not seem to have `lua.a` and `lua.dylib` as in the Xcode project.

I also have cinder downloaded but not sure how this project should include it. `LiveWare_Prefix.pch` seems to indicate it should be within the app?

**Comment by shawnlawson** [comment]
Yeah the instructions aren't so great yet. I believe I installed lua with Brew. They're hard-coded paths to /usr/local/lib  can be pointed to other locations if selected on the left and then using the info panel on the right in Xcode to choose a different location. may not actually need luarocks, I think I took all the rocks out.

For cinder, choose the liveware project on the left, then go to build settings. Way down the list is a user-defined cinder_path. Mine says /Users/lawsos2/liveCoding/Liveware/cinder_0.9.1_mac  Change this to be the path to your cinder. 

lemme know how those go.
